src/components/SideMenu/SideMenu.test.tsx

```diff
@@ -0,0 +1,68 @@
+import {
+  fireEvent,
+  getByTestId,
+  getByText,
+  render,
+  screen,
+} from "@testing-library/react";
+import AdminHomePage from "../../pages/AdminHomePage/AdminHomePage";
+import { MemoryRouter } from "react-router";
+import { routes } from "../../constants/routes";
+
+describe("Test side-menu", () => {
+  test("should render side menu on admin page", async () => {
+    render(
+      <MemoryRouter
+        initialEntries={[
+          {
+            pathname: routes.HOME,
+          },
+        ]}
+      >
+        <AdminHomePage />
+      </MemoryRouter>
+    );
+
+    const menuComponent = await screen.findByTestId(`menu-list`);
+
+    expect(menuComponent).toBeTruthy();
+  });
+
+  test("should change selected item on click", () => {
+    const { getByTestId } = render(
+      <MemoryRouter
+        initialEntries={[
+          {
+            pathname: routes.HOME,
+          },
+        ]}
+      >
+        <AdminHomePage />
+      </MemoryRouter>
+    );
+
+    const readerButton = getByTestId("reader-button");
+
+    fireEvent.click(readerButton);
```

Comment:
It would be better to use `userEvent` here, instead of `fireEvent`, because it simulates complete interactions, and not only single events. I know that it's not always used in other test, but we can start using it now for new ones as the better practice. See `AccountAvatar.test` for example usage.
https://testing-library.com/docs/user-event/intro/